core/src/Dscp/Crypto/Impl.hs

```diff
@@ -75,6 +80,27 @@ type PublicKey = AbstractPK SigScheme
 type SecretKey = AbstractSK SigScheme
 type Signature a = AbstractSig SigScheme a
 
+data DscpSigScheme
```

Comment:
Snowdrop is only allowed to be used in `witness`, and it'd be better if it's only inside `Dscp.Snowdrop`, so move this elsewhere pls.

Reply:
@astiunov please fix

Reply:
But it's already used in https://github.com/DisciplinaOU/disciplina/blob/release-alpha/core/src/Dscp/Util.hs#L62

Reply:
It's completely different. You should have put `OldestFirst` and `NewestFirst` in lootbox instead, but since you define it in sd we're forced to re-export it from our utility module.